### DISCUSSION TITLE
Automatic rounding up

### DISCUSSION THREAD

**Discussion by Keiishu** [firefly-iii/firefly-iii#6737] [Firefly III] [answered]
Hi!
My bank has a service called "Easy Save", which basically rounds up every transaction (not deposits nor withdrawals), and automatically transfers the difference on a saving account.
But I have no idea how I could insert this rule into my Firefly instance, and I don't want to double my work for every payment I make with my card.
As far as I know, we can't put mathematical operations in a rule action.

How could I achieve this?

**Comment by JC5** [comment, marked as answer]
Check out https://github.com/jc5/autosave

**Reply by Keiishu** [reply]
Thank you!
